docs/build/start-building.md

```diff
@@ -248,14 +248,21 @@ Push notifications can be a highly effective way to engage your users and increa
 
 Follow these guidelines to optimize your app’s performance.
 
-### Architecture
+### Use local storage
```

Comment:
WDYT of using the word `cache` instead of storage?

Reply:
Yes, sure thing, @rygine - thank you for taking a look! Updating now....